README.md

```diff
@@ -1,3 +1,8 @@
-# deskpro-styles
+deskpro-components
```

Comment:
I stopped short of changing the name in package.json, but it appears deskpro-styles was chosen when the project was new, and deskpro-components is the actual name now.